buildpacks/jvm-function-invoker/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+* Changed implementation to Rust
+* Upgrade libcnb to `0.1.3`
```

Comment:
```suggestion
* Changed implementation to Rust (relanded with upgrade to libcnb `0.1.3`)
```
(Just since as far as the changelog is concerned, it's a single rollout)